webpack.config.js

```diff
@@ -69,19 +68,6 @@ module.exports = env => {
     })
   );
 
-  if (env === 'sandbox') {
```

Comment:
This was removed because we now start the sandbox at the same time with the webpack process.